pages/linux/pvs.md

```diff
@@ -0,0 +1,31 @@
+# pvs
+
+> Display information about physical volumes.
+
+- Display information about physical volumes:
+
+`pvs`
+
+- Display non-physical volumes:
+
+`pvs -a`
+
+- Change default display to show more details:
+
+`pvs -v`
+
+- Display only specific fields:
+
+`pvs -o {{field_name_1}},{{field_name_2}}`
+
+- Append field to default display:
+
+`pvs -o +{{field_name}}`
+
+- Suppress heading line:
```

Comment:
```suggestion
- Suppress heading lines:
```
Maybe it's better like this?

Reply:
In `pvs` output, there's always one heading line. Man-page option `--noheadings` option too document it as 'line'.